go/exorelay/main.go

```diff
@@ -20,30 +20,33 @@ type Config struct {
 	Role string
 }
 
-// ExoRelay is the Go API to talk to Exocom
+// MessageOptions contains the configuration values for ExoRelay instances
+type MessageOptions struct {
+	Name       string
+	Payload    structs.MessagePayload
+	ResponseTo string
+}
+
+// ExoRelay is the low level Go API to talk to Exocom
 type ExoRelay struct {
-	config         Config
+	Config         Config
 	socket         *websocket.Conn
 	messageChannel chan structs.Message
 }
 
-// New creates a new ExoRelay instance
-func New(config Config) *ExoRelay {
-	return &ExoRelay{
-		config:         config,
-		messageChannel: make(chan structs.Message),
-	}
-}
-
 // Connect brings an ExoRelay instance online
 func (e *ExoRelay) Connect() error {
-	socket, err := websocket.Dial("ws://"+e.config.Host+":"+strconv.Itoa(e.config.Port), "", "origin:")
+	socket, err := websocket.Dial("ws://"+e.Config.Host+":"+strconv.Itoa(e.Config.Port), "", "origin:")
```

Comment:
might be easier to use a `sprintf` here rather than `strconv`.

However after talking with @hugobho today, I realized since we're running these services in docker locally for development, we can actually always run exocom on 80/443 which would mean not having to specify the port here. This seems more idiomatic since the port will be inferred from the protocol (`ws` or `wss`).

What are your thoughts @charlierudolph @kevgo?